detox/src/utils/logger.js

```diff
@@ -11,6 +14,9 @@ const temporaryPath = require('../artifacts/utils/temporaryPath');
 const argparse = require('./argparse');
 const customConsoleLogger = require('./customConsoleLogger');
 
+const LOG_DATE_FORMAT = "HH:MM:ss.l";
+const shortDateFormatter = (date) => dateFormat(date, LOG_DATE_FORMAT);
```

Comment:
Won't it be more efficient to:

```js
date.toTimeString().replace(/\s.*$/, '.' + String(date.getMilliseconds()).padStart(3, '0'))
```

instead of importing yet another package?

Reply:
Definitely. I will do that 👍🏻

Reply:
I was contemplating whether I would be better to have this for future usage. On 2nd thought, I'm inclined to say that since this hasn't troubled us this long, the odd of it being useful in the future are slim.

Reply:
I see what you mean. If we ever need more advanced date manipulation, we can add it later, no problem.

Reply:
@noomorph I've taken a rather more verbose approach, that is nevertheless aligned with your suggestion. Please have a glance.